src/overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h

```diff
@@ -4,7 +4,22 @@
 #include "global.h"
 
 // Debug
-void KaleidoScope_DrawDebugEditor(PlayState* play);
-void KaleidoScope_UpdateDebugEditor(PlayState* play);
+void KaleidoScope_DrawInventoryEditor(PlayState* play);
+void KaleidoScope_UpdateInventoryEditor(PlayState* play);
```

Comment:
We usually put function prototypes at the bottom of headers, unless you have something else in mind?